styles/stylesheet.css

```diff
@@ -0,0 +1,419 @@
+  top: auto;
+}
+
+.hidden:focus {
+  left: 0;
+}
+
+header {
+  width: 100%;
+  background-image: url('../images/header-image.jpg');
+  background-size: auto;
+  background-position: center center;
+  background-color: linear-gradient(to left, #d3d3d3, #3d3d3d, #083c60);
+  margin: 0;
+}
+
+
+.fas, .fa-shopping-cart {
+  float: right;
+  color: #ffffff;
+}
+
+.shopping {
+  display: flex;
+  flex-flow: column;
+  float: right;
+  margin: 0;
+}
+.checkout {
+  background-color: #81b741;
+  text-align: center;
+  border: 2px solid white;
+}
+
+.checkout-2 {
+  color: #ffffff;
+  text-align: center;
+}
+
+.logo-image {
+  width: 45%;
+  margin: 0;
+}
+
+h1 {
+  font-family: 'Domine', serif;
+  font-size: 2.5em;
+  font-style: italic;
+  color: #81b741;
+  border: none;
+  text-align: left;
+  width: auto;
+  margin: 0;
+}
+
+h2 {
+  font-family: 'Domine', serif;
+  font-size: 2em;
+  color: #083c60;
+  border: none;
+  margin: 0;
+}
+
+h3 {
+  font-family: 'Domine', serif;
+  font-size: 1.5em;
+  color: #083c60;
+  padding: .25em;
+  border: none;
+  margin: 0;
+}
+
+h4 {
+  font-family: 'Domine', serif;
+  font-size: 1.25em;
+  color: #083c60;
+  padding: .5em;
+  border: none;
+  margin:0;
+}
+
+a, div {
+  font-family: 'Roboto', sans-serif;
+  text-decoration: none;
+}
+
+p {
+  font-family: 'Roboto', sans-serif;
+  font-size: 1em;
+  color: #3d3d3d;
+  padding: .25em;
+  border: none;
+  margin: 0;
+}
+
+.cost {
+  font-weight: 900;
+}
+
+nav ul {
+  display: flex;
+  flex-flow: row wrap;
+  list-style: none;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1em;
+  color: #083c60;
+  background-color: #d3d3d3;
+  width: 100%;
+  padding: 2em;
+  border: none;
+  margin: 0;
+  text-align: center;
+}
+
+li a.menu-links {
+  color: #083c60;
+  margin: 0;
+  /*margin: 0 .5em .5em .5em;*/
+  border: none;
+  text-decoration: none;
+  text-align: center;
+}
+
+ul.my-list {
+  list-style: none;
+  color: #083c60;
+  font-size: 1em;
+}
+
+a:link {
+  color: #083c60;
+}
+
+a:visited {
+  color: #3d3d3d;
+}
+
+a:hover {
+  color: #81b741;
+}
+
+aside {
+  float: right;
+  display: flex;
+  flex-direction: column;
+  width: 18%;
+  margin-left: 2%;
+  margin-right: 5%;
+}
+
+section {
+  float: left;
+  display: inline-block;
+  width: 68%;
+  margin-left: 5%;
+  margin-right: 2%;
+}
+
+.centered-content-container {
+  display: inline-block;
+  float: left;
+  width: 70%;
+  text-align: center;
+  margin-left: 5%;
+}
+
+#main-content {
+  float: left;
+  display: block;
+  text-align: center;
+}
+
+.intro-text {
+  font-size: 1em;
+}
+
+.hug {
+  padding: .25em;
+  margin: 0.5em;
+  float: left;
+}
+
+.intro-text {
+  color: #494949;
+  text-align: center;
+}
+
+.grid-container, .grid-container-2 {
+  display: grid;
+  grid-gap: 1em 1em;
+  grid-template-columns: auto auto auto;
+  margin: 1em;
+}
+
+.card {
+  width: 100%;
+  height: auto;
+}
+
+.dog-1, .dog-2, .dog-3, .dog-4, .dog-5, .dog-6, .dog-7, .dog-8, .dog-9, .dog-10, .dog-11, .dog-12 {
```

Comment:
It might make more sense to have one .dog class instead of unique ones. Usually if you need to target them individually you use an #id tag instead of .class